streams/src/test/java/org/apache/kafka/streams/state/internals/CachingWindowStoreTest.java

```diff
@@ -329,6 +333,7 @@ public void shouldFlushEvictedItemsIntoUnderlyingStore() {
         assertEquals(added - 1, cache.size());
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
It seems like we can migrate away from the deprecated method in this test.

Reply:
removed